vagrant/scripts/goDeps.sh

```diff
@@ -1,5 +1,11 @@
 export GOPATH=/home/vagrant/go;
 export PATH=$PATH:/usr/local/go/bin;
+
+# Server dependencies.
 go get -d github.com/dancannon/gorethink;
 go get -d github.com/codegangsta/negroni;
 go get -d github.com/gorilla/mux;
+
+# Go command line tools.
+echo "export PATH=$PATH:$GOPATH/bin;" >> /home/vagrant/.profile;
```

Comment:
Is this supposed to be .bash_profile or .bashrc? Also doesn't line 2 handle this exact thing? PATH stuff tends to be a pain for me though so I could be completely off.